language/move-lang/tests/move_check/parser/spec_parsing_inside_fun.exp

```diff
@@ -15,6 +25,26 @@ error[E04003]: built-in operation not supported
    │         │          
    │         Found: '()'. But expected: 'u8', 'u64', 'u128'
 
+error[E04007]: incompatible types
+   ┌─ tests/move_check/parser/spec_parsing_inside_fun.move:33:9
+   │
+33 │         spec {} && spec {};
+   │         ^^^^^^^
+   │         │
+   │         Incompatible arguments to '&&'
+   │         Is not compatible with: 'bool'
+   │         The type: '()'
```

Comment:
The order of these is different from the order in the other comment (and out of order).